resources/lang/en/beatmaps.php

```diff
@@ -40,6 +40,7 @@
         'edited' => 'Last edited by :editor :update_time.',
         'kudosu_denied' => 'Denied from obtaining kudosu.',
         'message_placeholder' => 'Type here to post',
+        'message_placeholder_deleted_beatmap' => 'Posting on deleted difficulty is not allowed.',
```

Comment:
`"_xx_ is not allowed"` sounds a bit... blunt? Maybe something like `"This difficulty has been deleted so it may no longer be discussed."`